boards/stm32f746g-disco/board.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2021 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_stm32f746g-disco
+ * @{
+ *
+ * @file
+ * @brief       Board specific implementations for the STM32F746G-DISCO board
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+
+#include "board.h"
+#include "periph/gpio.h"
+
+void board_init(void)
+{
+}
```

Comment:
Do we need empty init functions?

Reply:
I tried to remove board.c and the build failed at link time. So I guess this is required.

Reply:
Ok!